tla-bmcmt/src/test/scala/at/forsyte/apalache/tla/bmcmt/TestSymbStateRewriterBool.scala

```diff
@@ -357,13 +357,13 @@ class TestSymbStateRewriterBool extends RewriterBase with TestingPredefs with Be
     assertUnsatOrExplain(rewriter, nextState)
   }
 
-  test("""SE-EX: \E x \in {1, 2}: y' = x ~~> 2 assignments, regression""") {
+  /** Jure, 9.12.19: Why should this throw? */
```

Comment:
It was throwing an exception for assignments that are inside non-scolemizable existentials. Since they are expanded into a disjunction, there is no way to assign a single value. Why does not it throw an exception anymore?

Reply:
After updating `QuantRule.isAssignmentInside` to check for the new syntax, the test now correctly throws.